nvaccess/nvda
# Support  the new Microsoft Quick input method for Chinese Traditional (Hong Kong)

### Link to issue number:
None.

### Summary of the issue:
Windows 10 introduces a newer version of the Microsoft Quick input method for Chinese Traditional (Hong Kong). these new version is all UI automation, integrated into the  windowsinternal composableshell_experiences textinput_inputapp.
Users expect Automatically reading all candidates in a list, including their number, and optional short character descriptions.

 ### Description of how this pull request fixes the issue:
Ensure that these UI automation candiate items and lists get the correct behaviour in NVDA by making use of our existing candidate behaviour NVDAObjects.

### Testing performed:
* In Windows 10 settings:
 * Add the chinese Traditional (Hong Kong) language
 * Add the Microsoft Quick input method as a keyboard to this language if not there already.
* In Notepad or another text editor:
 * Switch to chinese Traditional (Hong Kong) language with alt+shift
 * Ensure that you are in native input mode by tapping leftShift+rightShift until you are.
 * Type h then i. A candidate list should appear and NVDA should read it all. Moving the up and down arrows should speak the currently selected item.
 * Press 5 to choose the 5th candidate. A new Associated phrases list should appear and NVDA should automatically read all candidates.
 * Press shift+1 to choose the first assocated phrase and NVDA should announce that these symbols are typed into the document.
 
### Known Issues
None

### change log
New features:
- Added support for the new chinese Traditional Quick Input method in Windows 10.
